install/helm/agones/templates/NOTES.txt

```diff
@@ -1,10 +1,10 @@
 The Agones controller has been installed in the namespace {{ .Release.Namespace }}.
 
-You can watch the status by running 'kubectl --namespace {{ .Release.Namespace }} get pods -o wide -w {{ .Values.agones.serviceaccount.controller }}'
+You can get the status by running 'kubectl --namespace {{ .Release.Namespace }} get pods -o wide | grep {{ .Values.agones.serviceaccount.controller }}'
 
 Once ready you can create your first GameServer using our examples:
 
-'kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/simple-udp/gameserver.yaml'
+'kubectl create -f https://raw.githubusercontent.com/googleforgames/agones/master/examples/simple-udp/gameserver.yaml'
```

Comment:
Here's a fun question - should we be pointing people to master here. Probably not, since at some point it will be broken.

Since we're here, should we revamp this to remove all the example text, and point them to our online quickstart instead?

I figure short and sweet is likely better than all this text long term - I don't think I've ever read this, it scrolls right past when Agones installs.

WDYT?

Reply:
Yes, it is definitely better to have one source of truth. So I would add a link

Reply:
if we are in here, can we remove the markdown? It's a .txt file, I think it just looks weird?

Reply:
yes, md is not needed in here